src/meshlink.c

```diff
@@ -2644,6 +2663,18 @@ void meshlink_blacklist(meshlink_handle_t *mesh, meshlink_node_t *node) {
 		}
 	}
 
+	utcp_abort_all_connections(n->utcp);
```

Comment:
Did you add this function to UTCP already? If so, you should make a commit and a pull request for that as well.

Reply:
Guus, I think I'm not authorized to make changes. When I tried to push the code I got this error "fatal: remote error: access denied or repository not exported"